### ISSUE TITLE
Hi bro,

### ISSUE THREAD

**Issue by devilmp** [Mobeen22-creator/coursera-testi#1]
Hi, How can I contact you? I saw your comment via topic https://github.com/azerpas/nikeAPI-Py/issues/3 and got the same issue like you. Have you figured that out? Please share and discuss with the solution

This response:
{
"sec-cp-challenge": "true",
"provider": "crypto",
"branding_url_content": "%2fstatic%2fbot%2fchallenge%2findex.html",
"chlg_duration": 30
}

**Comment by Mobeen22-creator** [comment]
It due to cookies. It was working last month but suddenly it stop working.. You have bypass akamai protection

**Comment by Mobeen22-creator** [comment]
Decode the Akamai file stored in Nike public files:
https://www.nike.com/static/91698b6f882ti2091b8958ff21d04d86a
Once you understand its logic, check how to generate and to use _abck cookie on login.
Then it need to generate other cookies for using nike api..

**Comment by devilmp** [comment]
Hi bro,
I got to understand that file already. Have you passed the last error you got? it's because of static file _abck?
Can you clarify me on which stage you have to push the sensor data to get through the problem you posted?

Thanks

**Comment by Mobeen22-creator** [comment]
I was able to decode that file. Still I am looking for solution. 
I think you need to set cookies in nike checkout api.

**Comment by Mobeen22-creator** [comment]
Can you help me to decode that file?

**Comment by devilmp** [comment]
I'm in progress of it as well, as far as I can see the point come from bmak.sensor data. From that I think we can trace back on how it's created.  What I have done is using the cookie capture and token to process the checkout as it last 1 hour. But being stuck at the last step so I think the problem is not the cookie at all. If can't go through this then decode the bmak file seems meaningless

**Comment by devilmp** [comment]
it seems at last step checkout NIke did put some bot protection over the the query
{"sec-cp-challenge": "true","provider":"crypto","branding_url_content":"%2fstatic%2fbot%2fchallenge%2findex.html","chlg_duration":30}

**Comment by Mobeen22-creator** [comment]
Yes..Nike put bot protection last month..

**Comment by devilmp** [comment]
So since last month you was still be able to use api checkout normally right? I wonder how you can log in by api without cookies?

**Comment by Mobeen22-creator** [comment]
Yes, I was using checkout api normally last month..I can't able to login without cookies

**Comment by Mobeen22-creator** [comment]
Hi bro, I found a solution..

**Comment by cyphercodes96** [comment]
@Mobeen22-creator lmk if we can connect and talk about the solution you found and how we can best optimize it

**Comment by Mobeen22-creator** [comment]
Sure we can talk about solution

**Comment by Ladif1** [comment]
👀
